src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -9,6 +9,7 @@ import type { IntlShape } from 'react-intl';
 
 import LoadingIndicatorWrapper from '../../components/loading-indicator/LoadingIndicatorWrapper';
 import { Modal } from '../../components/modal';
+import type { SelectOptionProp } from '../../components/select-field/props';
```

Comment:
Should move type down with other type imports

Reply:
done